Library/Tracking/Koala.swift

```diff
@@ -2052,45 +2042,64 @@ public final class Koala {
   }
 }
 
+// MARK: - Project Properties
+
+// FIXME: remove the function below and replace with projectProperties
+
 private func properties(
   project: Project,
   loggedInUser: User?,
   prefix: String = "project_"
+) -> [String: Any] {
+  return projectProperties(from: project, loggedInUser: loggedInUser, prefix: prefix)
+}
+
+private func projectProperties(
+  from project: Project,
+  reward _: Reward? = nil,
+  backing _: Backing? = nil,
+  loggedInUser: User? = nil,
+  dateType: DateProtocol.Type = AppEnvironment.current.dateType,
+  calendar: Calendar = AppEnvironment.current.calendar,
+  prefix: String = "project_"
 ) -> [String: Any] {
   var props: [String: Any] = [:]
 
   props["backers_count"] = project.stats.backersCount
+  props["category"] = project.category.name
   props["country"] = project.country.countryCode
+  props["comments_count"] = project.stats.commentsCount
   props["currency"] = project.country.currencyCode
+  props["creator_uid"] = project.creator.id
+  props["deadline"] = project.dates.deadline
   props["goal"] = project.stats.goal
-  props["pid"] = project.id
+  props["launched_at"] = project.dates.launchedAt
+  props["location"] = project.location.name
   props["name"] = project.name
-  props["pledged"] = project.stats.pledged
+  props["pid"] = project.id
+  props["parent_category"] = project.category.parent?.name
   props["percent_raised"] = project.stats.fundingProgress
-  props["has_video"] = project.video != nil
+  props["pledged"] = project.stats.pledged
   props["state"] = project.state.rawValue
-  props["update_count"] = project.stats.updatesCount
```

Comment:
Seems strange to remove `update_count`?

Reply:
Ah seems I missed that that one's been renamed to `updates_count`! Good 👁